### PR TITLE
Fix link display

### PR DESCRIPTION
**Pull request by ptibogxiv**

### CODE REVIEW

File: htdocs/stripe/transaction.php [hunk 1]
```diff
@@ -80,26 +82,25 @@
...
-        print '<input type="hidden" name="optioncss" value="' . $optioncss . '">';
-    }
-	print '<input type="hidden" name="token" value="' . $_SESSION['newtoken'] . '">';
-	print '<input type="hidden" name="formfilteraction" id="formfilteraction" value="list">';
-	print '<input type="hidden" name="action" value="list">';
-	print '<input type="hidden" name="sortfield" value="' . $sortfield . '">';
-	print '<input type="hidden" name="sortorder" value="' . $sortorder . '">';
-	print '<input type="hidden" name="page" value="' . $page . '">';
+	print '<FORM method="POST" action="' . $_SERVER["PHP_SELF"] . '">';
+	if ($optioncss != '')
```
Comment (frederic34) [inline htdocs/stripe/transaction.php]: please keep lowercase

File: htdocs/stripe/transaction.php [hunk 2]
```diff
@@ -122,14 +123,14 @@
 		$txn = \Stripe\BalanceTransaction::all(array("limit" => $limit));
 	}
 
-	foreach ($txn->data as $txn)
+	foreach ($txn->data as $txn)
```
Comment (frederic34) [inline htdocs/stripe/transaction.php]: please do not add whitespace

**Comment by ptibogxiv** [comment]
@eldy PR fix :)

**Comment by frederic34** [comment]
👍

**Pull request merged**